src/app/pages/administration/jobs/system-triggers-view.jsx

```diff
@@ -41,84 +45,139 @@ const useStyles = makeStyles({
+
+  const getAction = (item) => {
+    if (item.triggerScheduleType === 'ONE_TIME') {
+      return (
+        <IconButton
+          onClick={() => confirmDelete(item)}
+          variant="contained"
+          color="primary"
+          aria-label={`Delete Job ${item.name}`}
+        >
+          <DeleteIcon />
+        </IconButton>
+      );
+    }
+    return null;
+  };
+
   useEffect(() => {
     setTriggers(props.triggers
       .sort((a, b) => (a.nextRunDate - b.nextRunDate))
       .map((trigger) => ({
         ...trigger,
+        action: getAction(trigger, dispatch),
       })));
-  }, [props.triggers]); // eslint-disable-line react/destructuring-assignment
+  }, [props.triggers, dispatch]); // eslint-disable-line react/destructuring-assignment
 
   return (
-    <Card>
-      <CardHeader title="Currently Scheduled System Jobs" />
-      <CardContent>
-        <>
-          { (triggers.length === 0)
-            && (
-              <Typography className={classes.noResultsContainer}>
-                No results found
-              </Typography>
-            )}
-          { triggers.length > 0
-            && (
-              <TableContainer className={classes.container} component={Paper}>
-                <Table
-                  aria-label="Scheduled System Jobs table"
-                >
-                  <ChplSortableHeaders
-                    headers={headers}
-                    onTableSort={() => {}}
-                    orderBy="name"
-                    order="asc"
-                    stickyHeader
-                  />
-                  <TableBody>
-                    { triggers
-                      .map((item) => (
-                        <TableRow key={item.nextRunDate}>
-                          <TableCell className={classes.firstColumn}>
-                            { item.name }
-                          </TableCell>
-                          <TableCell>
-                            { item.description }
-                          </TableCell>
-                          <TableCell>
-                            { item.nextRunDate
-                              ? (
-                                <Moment
-                                  fromNow
-                                  withTitle
-                                  titleFormat="DD MMM yyyy, h:mm a"
-                                >
-                                  {item.nextRunDate}
-                                </Moment>
-                              ) : (
-                                <>In Progress</>
-                              )}
-                          </TableCell>
-                          <TableCell>
-                            { item.triggerScheduleType }
-                          </TableCell>
-                        </TableRow>
-                      ))}
-                  </TableBody>
-                </Table>
-              </TableContainer>
-            )}
-        </>
-      </CardContent>
-    </Card>
+    <div>
```

Comment:
This should be a React.Fragment instead of a <div> element. The reason is that DIVs have layout meaning, while the fragment things are just "grouping" elements

```suggestion
    <>
```